test/src/test/java/jenkins/security/apitoken/LegacyApiTokenAdministrativeMonitorTest.java

```diff
@@ -76,6 +89,40 @@ public void isActive() throws Exception {
         assertTrue(monitor.isActivated());
     }
     
+    @Test
+    @Issue("JENKINS-52441")
+    public void takeCareOfUserWithIdNull() throws Exception {
+        ApiTokenPropertyConfiguration config = ApiTokenPropertyConfiguration.get();
+        config.setCreationOfLegacyTokenEnabled(true);
+        config.setTokenGenerationOnCreationEnabled(false);
+        
+        // user created without legacy token
+        User user = User.getById("null", true);
+        ApiTokenProperty apiTokenProperty = user.getProperty(ApiTokenProperty.class);
+        assertFalse(apiTokenProperty.hasLegacyToken());
+        
+        LegacyApiTokenAdministrativeMonitor monitor = j.jenkins.getExtensionList(AdministrativeMonitor.class).get(LegacyApiTokenAdministrativeMonitor.class);
+        assertFalse(monitor.isActivated());
+        
+        apiTokenProperty.changeApiToken();
+        assertTrue(monitor.isActivated());
+    
+        {//revoke the legacy token
```

Comment:
Curiosity: Why the brackets { } ? It creates a different scope of variables, affects the code. Is it necessary? Isn't it better to use only comments?

Reply:
It's a common practice in tests to provide comment on block of code without the requirement to create a method. It's especially useful when you use lots of variables and an method extraction could be less readable. Also interesting to create a new scope of variable to avoid having "user1", "user2", "user3" etc. Look at `listOfUserWithLegacyTokenIsCorrect` for example.

Reply:
Ok thanks. I said this because there wasn't collisions on variable names, so I didn't consider it necessary. In any case, It doesn't block my approval